autoload/previm/assets.vim

```diff
@@ -122,6 +122,7 @@ let s:source_map = [
 \        'path': '_/js/lib/mermaid.min.js',
 \        'url': 'https://cdn.jsdelivr.net/npm/mermaid@latest/dist/mermaid.min.js',
 \        'code': [
+\          'mermaid.initialize({startOnLoad: false});',
 \          'mermaid.init();',
```

Comment:
:question:

Calling `mermaid.init()` immediately after `mermaid.initialize({startOnLoad: false})` doesn't make sense to me.

I'm not sure but it seems using `mermaid.init()` itself is already deprecated?
https://github.com/mermaidjs/mermaidjs.github.io/blob/master/usage.md#using-the-mermaidinit-call

Reply:
Indeed.
But without either of these two lines, Mermaid rendering fails...

Perhaps I misunderstand the way to initialization.
If you know the correct way, please fix it...

Reply:
Now I found: it should be `mermaid.run()`
(cf. <https://mermaid.js.org/config/usage.html#using-mermaid-run>).

Reply:
Although, `mermaid.run()` requires mermaid.js v10, at least.

Updated mermaid.js to the latest v10.7.0 together.